### ISSUE TITLE
Application does not run in root.

### ISSUE THREAD

**Issue by JuanRodenas** [gollum/gollum#1870]
Hello.
I request that the application does not run in root. Otherwise, use the UID and GID variables. I have tried to compile the app, it compiles but when accessing the container it gives me error, can you help me to make it run on user?

I attach the error.

![IMG_20221111_013821_244.png](https://user-images.githubusercontent.com/95724863/201235712-b35d339e-ed90-4851-bcc7-6df9e279f5d0.png)

Best regards,
Juan

**Comment by dometto** [comment]
You're right that running Docker processes as root is [not best practice](https://www.howtogeek.com/devops/why-processes-in-docker-containers-shouldnt-run-as-root/). Happy to accept a PR changing the Dockerfile!

**Comment by JuanRodenas** [comment]
Thank you for responding to my request.
Perfect, give me a few days, let me test that everything works and I'll make you a PR request.
Expect to hear from me soon.
Best regards, juan
